UI/data/locale/en-US.ini

```diff
@@ -630,6 +631,7 @@ Basic.Settings.Output.Adv.Rescale="Rescale Output"
 Basic.Settings.Output.Adv.AudioTrack="Audio Track"
 Basic.Settings.Output.Adv.Streaming="Streaming"
 Basic.Settings.Output.Adv.ApplyServiceSettings="Enforce streaming service encoder settings"
+Basic.Settings.Output.Adv.DynamicalBitrate="Enable Dynamical Variable Bitrate (not supported for nvenc)"
```

Comment:
I think "Dynamical Variable Bitrate" is a bit too hard for a user to understand what it does.

I would suggest "Auto adjust bitrate for network conditions" or "Auto manage bitrate if congestion detected" or something along those lines.

Reply:
thanks R1CH, you're right, will change to what you suggest.
Dodgepong suggested to add  an indicator at the bottom when bitrate switching is active.
Wondering also if it would be useful to add some user control on the bitrate decrease and increase rate (that would be in Advanced mode obviously).